### ISSUE TITLE
code for edge extension 

### ISSUE THREAD

**Issue by fym35** [tosdr/tosdr-ie#2]
The code to make a edge extension is here
https://docs.microsoft.com/en-us/microsoft-edge/extensions/guides/creating-an-extension

**Comment by michielbdejong** [comment]
Yes, I actually submitted one to the Edge extension store, using the code from https://github.com/tosdr/browser-extensions

**Comment by abhihossain** [comment]
🤩 I just earned $242 with RewardsFeed under 30 minutes... Sounds too good? You can try it for yourself! 🤑 Sign up within 24h to get a FREE $25 bonus https://rewardshare.net/?coupon=00a-abhiho

**Comment by michielbdejong** [comment]
cc @JustinBack
